internal/query/user.go

```diff
@@ -920,6 +929,7 @@ func prepareUserQuery(ctx context.Context, db prepareDatabase) (sq.SelectBuilder
 			HumanIsEmailVerifiedCol.identifier(),
 			HumanPhoneCol.identifier(),
 			HumanIsPhoneVerifiedCol.identifier(),
+			HumanPasswordChangeRequiredCol.identifier(),
```

Comment:
what's the reason we only return the field on a single users and not if we query for a list of users?

Reply:
Don't know why but missed to add it there, added with additional integration test